spec/reek/smell_detectors/feature_envy_spec.rb

```diff
@@ -292,4 +292,61 @@ def bravo(charlie)
 
     expect(src).not_to reek_of(:FeatureEnvy)
   end
+
+  it 'does not reports if supressed with a preceding code comment' do
+    src = <<-RUBY
+      class Alfa
+        # :reek:FeatureEnvy
+        def bravo(charlie)
+          (charlie.delta - charlie.echo) * foxtrot
+        end
+      end
+    RUBY
+
+    expect(src).not_to reek_of(:FeatureEnvy)
+  end
+
+  it 'reports when decreasing max_calls value with a comment' do
+    src = <<-RUBY
+      class Alfa
+        # :reek:FeatureEnvy { max_calls: 1 }
```

Comment:
Maybe you can use an example that would not be smelly if max_calls was not decreased?

Reply:
I'm actually not sure if this detector ever triggers with fewer than 2 calls ... 🤔

Reply:
You are  right I removed the smelly test haha